.github/workflows/PR-build.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches: 
       - main*
+      - feature*
```

Comment:
Are we keeping this for future feature branches? Otherwise we should probably remove

Reply:
Yes we are because we may want to feature branch other things in the future.